pom.xml

```diff
@@ -23,9 +23,9 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<main.dir>${basedir}</main.dir>
 		<java.version>11</java.version>
-		<guice.version>5.1.0</guice.version>
+		<guice.version>7.0.0</guice.version>
 		<querydsl.version>5.0.0</querydsl.version>
-		<hibernate.version>5.6.15.Final</hibernate.version>
+		<hibernate.version>6.3.1.Final</hibernate.version>
```

Comment:
Moving to Jakarta does not require to upgrade to Hibernate v6 

We can move to [hibernate-core-jakarta](https://in.relation.to/2021/06/02/hibernate-orm-550-final-release/)